packages/typespec-autorest-canonical/src/emitter.ts

```diff
@@ -144,12 +143,7 @@ function resolveOutputFile(
       multipleServices || azureResourceProviderFolder
         ? getNamespaceFullName(service.type)
         : undefined,
-    "version-status": azureResourceProviderFolder
-      ? version?.includes("preview")
-        ? "preview"
-        : "stable"
-      : undefined,
-    version,
+    version: canonicalVersion,
```

Comment:
you can remove that now

Reply:
removed version parameter and update the template config